backend-shared/config/db/application_test.go

```diff
@@ -41,6 +43,8 @@ var _ = Describe("Application Test", func() {
 
 		err = dbq.GetApplicationById(ctx, &applicationget)
 		Expect(err).To(BeNil())
+		Expect(applicationput.Created_on).To(BeAssignableToTypeOf(timestamp))
```

Comment:
Rather than checking if it's assignable to a timestamp, lets instead check if the time field is within the last 5 minutes. Something like this:
```
Expect(applicationput.Created_on.After(time.Now().Add(time.Minute*-5))).To(BeTrue(), "Create on should be within the last 5 minutes")
```
(feel free to check my logic in the statement, I haven't tested it. )

Presuming you agree, can you you make the change in all the tests where we are are doing a create, then a get?